frontend/src/index.css

```diff
@@ -14,7 +14,7 @@ body,
 #root {
   width: 100%;
   height: 100%;
-  background-color: #f8fafc;
+  background-color: #fff;
```

Comment:
Do we want to make this change here? Wouldn't this affect all pages? Not just the play canvas?